app/app/src/main/kotlin/com/hedvig/app/feature/referrals/ui/redeemcode/RedeemCodeViewModel.kt

```diff
@@ -15,7 +15,7 @@ import kotlinx.coroutines.launch
 
 class RedeemCodeViewModel(
   private val quoteCartId: QuoteCartId?,
-  private val referralsRepository: ForeverRepository,
+  private val referralsRepository: ForeverRepositoryImpl,
```

Comment:
This could still be ForeverRepository right?

Reply:
Thank you, yes!